context/AddContext.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+interface IAddContext {
+  additional_fields: Array<boolean>;
+  setAdditionalFields: (fields: Array<boolean>) => void;
+}
+
+const initialValues = {
+  additional_fields : [false, false, false, false, false],
```

Comment:
please, change to mo clean sense as { }

Reply:
This array used for checking checkbox status only. Refactored for using in common cases